### PR TITLE
Add APIs for serverless

### PR DESCRIPTION
**Pull request by viseshrp**
Added APIs to use ADR in a serverless manner. This unfortunately requires importing from the installation, otherwise this has to co-exist with the core.

### CODE REVIEW

File: src/ansys/dynamicreporting/core/adr.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    InvalidAnsysPath,
+    InvalidPath,
+    StaticFilesCollectionError,
+)
+
+
+class ADR:
+    """
+    >> from ansys.dynamicreporting.core import ADR, Table
+    >> opts = { "CEI_NEXUS_DEBUG" : "0", "CEI_NEXUS_SECRET_KEY":"", "CEI_NEXUS_LOCAL_DB_DIR":r"C:\\cygwin64\\home\vrajendr\\ogdocex" }
```
Comment (randallfrank) [inline src/ansys/dynamicreporting/core/adr.py:24]: LOL, do you want your login name in a public repo?  @viseshrp

File: src/ansys/dynamicreporting/core/adr.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        static_directory: str = None,
+        opts: dict = None,
+        request: HttpRequest = None,
+        logfile: str = None,
+    ) -> None:
+        self._db_directory = None
+        self._media_directory = None
+        self._static_directory = None
+
+        if ansys_installation is None:
```
Comment (randallfrank) [inline src/ansys/dynamicreporting/core/adr.py:45]: @viseshrp In PyEnSight, there is a staticmethod in locallauncher.py that we rather like for this.  get_cei_install_directory().   Tends to allow most folks to ignore the install path.

**Review by randallfrank — APPROVED** [review]

File: src/ansys/dynamicreporting/core/adr.py (hunk 1 above)
Comment (viseshrp) [reply src/ansys/dynamicreporting/core/adr.py]: lol, thanks

File: src/ansys/dynamicreporting/core/adr.py (hunk 2 above)
Comment (viseshrp) [reply src/ansys/dynamicreporting/core/adr.py]: done, thanks

**Review by randallfrank — APPROVED** [review]

**Pull request merged**